spec/integration/aws_lambda_examples_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+# encoding: utf-8
+
+require "spec_helper"
+
+describe "AWS Lambda examples in Ruby" do
+  require_aws_auth
+
+  it "shares the client" do
+    # Start AWS Lambda Example 1
+
+    # Import the dependency.
+    require "mongo"
+
+    # Create a Mongo::Client instance.
+    # CRITICAL: You must create client outside the handler so that the client
+    # can be reused across function invocations.
+    client = Mongo::Client.new(ENV["MONGODB_URI"])
+
+    def lambda_handler(event:, context:)
+      # Use the client to return the name of the connected database.
+      client.database.name
+    end
+
+    # End AWS Lambda Example 1
+
+    client.close
+  end
+
+  it "to the deployment using AWS IAM authentication" do
```

Comment:
I'm not sure what this test string actually means.

Reply:
I can't find where these examples are actually specified but I'm thinking this should be a `context` with something like "when using AWS IAM authentication" with the example under `it` under the context.

Reply:
The examples are specified here - https://www.mongodb.com/docs/atlas/manage-connections-aws-lambda/
and in the scope document:

> The scope of these code examples is limited to the instructions on:
> * How to share the client (“Example 1”. See first snippet in [this](https://www.mongodb.com/docs/atlas/manage-connections-aws-lambda/#connection-examples) section for structure)
> * How to connect to the deployment using AWS IAM authentication (“Example 2”. Same structure as “Example 1”, but see the blue note box in the [docs](https://www.mongodb.com/docs/atlas/manage-connections-aws-lambda) for which lines need to be replaced to work with AWS IAM auth)